### ISSUE TITLE
Update regression tests for MN4

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#252]
In GitLab by @ jlope2 on Mar 31, 2017, 16:07

Since MN3 is down, the current MN tests have no sense.

A good idea could be update them when the new MN be available.

@mcastril @pablohe @kserradell

**Comment by kinow** [comment]
In GitLab by @dmanubens on Jun 30, 2017, 11:56

let's go

**Comment by kinow** [comment]
In GitLab by @dmanubens on Jun 30, 2017, 11:56

Started branch [`252-update-regression-tests-for-mn4`](https://earth.bsc.es/gitlab/es/autosubmit/compare/master...252-update-regression-tests-for-mn4)

**Comment by kinow** [comment]
In GitLab by @dmanubens on Jun 30, 2017, 18:05

good news is that mn_with_paramiko regression tests works
However when defining a `QUEUE` it does not work.
Message from support:
```
Vemos que has estado lanzando diversos jobs especificando en el Jobscript la partición debug. Mediante la constraint:

#SBATCH -p debug

Parece que estos submits se estan realizando en gran número, con algún método automatizado que nos esta generando diversos problemas en SLURM.

Los jobs han estado fallando debido a que no entiende dicha constraint, para lanzar a debug deberias sustituirla por:

 #SBATCH --qos=debug 

```
`qos` was not working for `moore` so more interaction with support will be needed.

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Jun 30, 2017, 19:05

Hi @dmanubens, 

If it hasn't, you will need to implement the proper function in the SlurmHeader and then call it from the ParamikoPlatform (line 409 and below). 

I think you have to decide a "common" attribute for the '''--qos" directive (quality of service), AFAIK the queue is commonly compared with the Account in the Slurm queues. 

Regards!

**Comment by kinow** [comment]
In GitLab by @dmanubens on Jul 19, 2017, 13:29

Hi @ jlope2 good to see you there! Sorry for not responding but we've been busy with ECMWF auto-ecearth primavera stuff... I'll look into MN4 as soon as I can! merci for your help

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Dec 4, 2018, 12:36

Solved in #271

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Dec 4, 2018, 12:36

closed
